README.md

```diff
@@ -0,0 +1,33 @@
+# quest-template
+
+## Quest Details 
+#### title: 
+#### level: 
+#### skills:
+#### dependencies: 
+
+
+## Overview 
+Explain what the quest is about and describe the main task the user needs to perform in this quest. 
+
+
+## Outline
+Describe each step in the quest. 
+- Step 1 [step id]: 
+- Step 2 [step id]:
+- ...
+
+
+## Textbook solution
+For each step, describle all actions the user needs to perform to complete the step, including links to PRs as they would need to be written to pass checks. 
+Before submitting your quest for review, test your quest in snack and check the instructions you wrote. Make sure that by following these instructions, you are able to successfully complete the quest.  
+### Instructions for completing the quest: 
+#### Step 1 [step id]: 
```

Comment:
I don't think that the solution to steps should break down into couple of items, since it should be simple and probably be solved with one step

Reply:
But a step needs to end with a verifiable interaction. Even when you say, "do this, and let me know when you're done", that's two actions in a single step.

Reply:
in most cases it will be one or 2 things to do, putting steps here will make people thing that they need to build it in a way that they need to do multiple steps